### PR TITLE
Merge 24.09 release branches

### PR DESCRIPTION
**Pull request by mmilata**
Merges `release/24.09` as well as `release/translations/24.09` into main.

Please merge using `git merge --ff-only` & push manually so that the merge commit is preserved. Previous release: #4114

### CODE REVIEW

**Comment by prusnak** [comment]
$ git merge --ff-only mmilata/merge-release-24.09
fatal: Not possible to fast-forward, aborting.

**Comment by mmilata** [comment]
Try again please?

File: core/src/apps/ethereum/definitions_constants.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     b"\xb8\xd2\xb2\x1d\xe2\x71\x24\xf0\x51\x1f\x90\x3a\xe7\xe6\x0e\x07\x96\x18\x10\xa0\xb8\xf2\x8e\xa7\x55\xfa\x50\x36\x7a\x8a\x2b\x8b",
 )
 
-MIN_DATA_VERSION = 1720519386
+MIN_DATA_VERSION = 1724682490
```
Comment (obrusvit) [inline core/src/apps/ethereum/definitions_constants.py:12]: Acknowledging that this timestamp is a little bit older than the current latest (`1725965878`).

**Review by obrusvit — APPROVED** [review]
utACK
- checked the changelog entries.

**Pull request merged**